addons/web/static/src/js/core/utils.js

```diff
@@ -500,6 +500,18 @@ var utils = {
     isComponent: function (value) {
         return value.prototype instanceof owl.Component;
     },
+    /**
+     * Checks if a keyboard event concerns
+     * the numpad decimal separator key
+     * while the NumLock modifier state is active.
+     * 
+     * @param {KeyboardEvent} ev
+     * @returns {boolean}
+     */
+    isNumpadDecimalSeparatorKey(ev) {
+        return ev.getModifierState('NumLock')
+            && (ev.code !== undefined && ev.code === 'NumpadDecimal');
```

Comment:
check on ev.code twice, and mutually exclusive ?

Reply:
My bad